coupons/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db          import models
+
+from restaurants.models import Restaurant
+
+class Coupon(models.Model):
+    name           = models.CharField(max_length=40)
+    restaurant_id = models.ForeignKey('Restaurant', on_delete=models.SET_NULL, db_column='restaurant_id', null=True)
```

Comment:
restaurant column 과 db_column 으로 지정된 column 이 동일하기 떄문에 굳이 db_column='restaurant_id' 를 명시하지 않아도 괜찮을 것 같습니다. 어떻게 생각하시나요?
```suggestion
    restaurant = models.ForeignKey('Restaurant', on_delete=models.SET_NULL, null=True)
```

Reply:
통일하려고 다 붙였는데 없어도 될 것 같습니다 !